### PR TITLE
xplat mdatp docs updates

### PR DESCRIPTION
**Pull request by maximvelichko**
- macOS Intune - modern workflow for app configuration
- macOS Intune - cleaning up profiles
- macOS config - clarifications on exclusions
- Linux config - clarification on exclusions

### CODE REVIEW

**Comment by maximvelichko** [comment]
@AirTudor - please review when you have time

**Comment by mjcaparas** [comment]
Will wait for @AirTudor review and approval.

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md [hunk 1]
```diff
@@ -104,105 +80,110 @@ Download the installation and onboarding packages from Microsoft Defender Securi
       inflating: jamf/WindowsDefenderATPOnboarding.plist
     ```
 
-8. Make IntuneAppUtil an executable:
+## Create System Configuration profiles
 
-    ```bash
-    chmod +x IntuneAppUtil
-    ```
+The next step is to create system configuration profiles that Defender needs.
```
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Full product name would be better than just `Defender`

Comment (AirTudor) [reply microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Same in other places, I don't believe we refer to our product just as `Defender`

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md [hunk 2]
```diff
@@ -104,105 +80,110 @@ Download the installation and onboarding packages from Microsoft Defender Securi
...
 
-## Client device setup
+1. Choose a name for the profile, e.g. "MDATP onboarding for macOS". Click **Next**.
 
-You don't need any special provisioning for a Mac device beyond a standard [Company Portal installation](https://docs.microsoft.com/intune-user-help/enroll-your-device-in-intune-macos-cp).
+    ![Custom Configuration Profile - name](images/mdatp-6-systemconfigurationprofiles-2.png)
 
-1. Confirm device management.
+1. Choose a name for the configuration profile name, e.g. "MDATP onboarding for macOS".
+1. Select intune/WindowsDefenderATPOnboarding.xml that you extracted from the onboarding package above as Configuration profile file.
```
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Minor: I believe this should either be lower-case `c`, or upper-case `P` (i.e. configuration profile / Configuration Profile). Having just Configuration capitalized looks a bit confusing

File: microsoft-365/security/defender-endpoint/mac-preferences.md [hunk 3]
```diff
@@ -132,6 +132,27 @@ Specify content excluded from being scanned by full file path.
 | **Possible values** | valid paths |
 | **Comments** | Applicable only if *$type* is *excludedPath* |
 
+## Supported exclusion types
+
+The follow table shows the exclusion types supported by Defender for Endpoint on Linux.
```
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-preferences.md]: `Mac` instead

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md [hunk 4]
```diff
@@ -104,105 +80,110 @@ Download the installation and onboarding packages from Microsoft Defender Securi
...
+> Apple Silicon (M1) devices do not support KEXT. Installation of a configuration profile consisting KEXT policies will fail on these devices.
+
+1. Select **Create Profile** under **Configuration Profiles**
+1. Select **Platform**=**macOS**, **Profile type**=**Templates**. **Template name**=**Extensions**. Click **Create**.
+1. In the **Basics** tab, give a name to this new profile.
+1. In the **Configuration settings** tab, expand **Kernel Extensions**.
+1. Set **Team identifier** to **UBF8T346G9** and click Next.
 
     Bundle identifier         | Team identifier
     --------------------------|----------------
```
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: This should probably be removed from under `Kernel Extensions` (these bundles are for system extensions instead)

**Review by AirTudor — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by maximvelichko** [comment]
@mjcaparas , can you complete/merge it?

**Pull request merged**